### ISSUE TITLE
highlight color in nord & everforest themes is not visible

### ISSUE THREAD

**Issue by ardentperf** [basecamp/omakub#358]
i'm not sure if the zellij themes are from omakub or upstream, and couldn't figure this out quickly. i like the nord theme however one issue is that when i select text with my mouse in zellij, the selection is not visible at all.  similar problem with everforest. doesn't seem to be an issue on any of the other themes. really makes copy-and-paste almost unusable in zellij... probably an easy fix but does someone here understand the zellij themes well enough to know what change is needed?

BTW - in the meantime, simple enough workaround is to manually edit `$HOME/.config/zellij/config.kdl` and set the zellij theme to a different one (after selecting nord or everforest from the omakub menu)

**Comment by florentdestremau** [comment]
it's weird but the text selection is meant to be used with the SHIFT key.

**Comment by ardentperf** [comment]
unfortunately the SHIFT copy thing does not work for multiline selects because it space-pads everything and add the zellij border. this makes copy/paste pretty unusable for multiline, which is sortof a dealbreaker for a lot of real work.

**Comment by florentdestremau** [comment]
I agree, for this you can toggle on/off the border with Ctrl + P → Z.

See #105 for the exact same discussion
